Recrutify/Recrutify.Services/Validators/CreateCandidateValidator.cs

```diff
@@ -1,5 +1,9 @@
-﻿using System.Linq;
```

Comment:
Points 1, 5 from trellis task are not fixed

Reply:
done

Reply:
1 point is not done

Reply:
Test next situation: create candidate with contacts : [null] and see what will happen

Reply:
done